pkgs/tools/virtualization/kubevirt/virtctl.nix

```diff
@@ -0,0 +1,47 @@
+{ buildGo117Module
+, fetchFromGitHub
+, lib
+, testers
+, virtctl
+}:
+
+buildGo117Module rec {
+  pname = "virtctl";
+  version = "0.52.0";
+  strictDeps = true;
+
+  src = fetchFromGitHub {
+    owner = "kubevirt";
+    repo = "kubevirt";
+    rev = "v${version}";
+    sha256 = "0ldc13nypbk3r9ihbdfibpyvzyzz4m68zzb7g6cq551ks9qdiyma";
+  };
+
+  vendorSha256 = null;
+
+  subPackages = [ "cmd/virtctl" ];
+
+  tags = [ "selinux" ];
+
+  ldflags = let
+    gitCommit = "836afc8c556206fc7715615ddb6516d76f4240db";
+  in [
+    "-X kubevirt.io/client-go/version.gitCommit=${gitCommit}"
+    "-X kubevirt.io/client-go/version.gitTreeState=clean"
+    "-X kubevirt.io/client-go/version.gitVersion=v${version}"
+  ];
```

Comment:
```suggestion
  ldflags = [
    "-X kubevirt.io/client-go/version.gitCommit=v${version}"
    "-X kubevirt.io/client-go/version.gitTreeState=clean"
    "-X kubevirt.io/client-go/version.gitVersion=v${version}"
  ];
```

Reply:
I like this, cause it's less maintenance.

It's not consistent with the officially released precompiled binary by KubeVirt. Should I apply this change anyway?

Reply:
> It's not consistent with the officially released precompiled binary by KubeVirt. Should I apply this change anyway?

Fine for me, I don't really care about the commit personally.